src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -741,6 +770,11 @@ struct jit_uni_eltwise_generic : public jit_uni_eltwise_kernel, public jit_gener
     }
 
     inline void load_scalar(Xmm xmm_src, const Xbyak::Address &op, Precision src_prc, Precision dst_prc) {
+        if (src_prc == dst_prc) {
+            uni_vmovss(xmm_src, op);
```

Comment:
vmovss always loads 4bytes from memory. This may lead to segfaults for 1/2 bytes input precisions.

Reply:
fixed, please explore and review, thanks!